src/app.js

```diff
@@ -0,0 +1,75 @@
+import { CONSTANTS } from './modules/constants';
+import { addTodo } from './modules/todo';
+import { renderTodoList } from './modules/dom';
+
+function init() {
+  const root = document.getElementById('root');
+
+  const today = new Date().toLocaleDateString('ko-KR', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+    weekday: 'long',
+  });
```

Comment:
개인적인 생각이지만.. 이미 다른 부분에서 모듈화를 잘 해놓으셨기 때문에 해당 날짜 관련 로직도 분리해서 쓰면 어떨까 싶네요. 
```suggestion
 export function getFormattedDate() {
  return new Date().toLocaleDateString('ko-KR', {
    year: 'numeric',
    month: 'long',
    day: 'numeric',
    weekday: 'long',
  });
}
```

이런 식으로 약간 바꾼 다음에 modules 폴더에 파일로 분리해서 넣고, app.js에서 임포트해서 `const today = getFormattedDate();` 이런 식으로 사용해도 좋을 것 같아요.

Reply:
넵 감사합니다! 저 부분도 코드 작성하는 과정에서 대충 그냥 넘어갔던 것 같아서 더 수정하도록 하겠습니다